formats/dot/README.md

```diff
@@ -0,0 +1,9 @@
+# formats/dot
+
+## License
+
+The source code and any original content of the formats/dot directory is released into the [public domain](https://creativecommons.org/publicdomain/zero/1.0/).
```

Comment:
I think this is not quite right. While the effect is like the intention of releasing the code into the public domain, it is actually a non-obliging license, I would say something like "The source code and any original content of the formats/dot directory is released under [Public Domain Dedication](https://creativecommons.org/publicdomain/zero/1.0/)." This includes the notion that is intended but avoids possible confusion with actual public domain, which may not be possible in some places.